src/main/java/com/code_intelligence/jazzer/driver/Opt.java

```diff
@@ -141,6 +141,17 @@ public final class Opt {
   public static final boolean isAndroid =
       boolSetting("android", false, "Jazzer is running on Android");
 
+  public static final String androidBootclassJarPath = stringSetting("android_bootclass_jar_path",
+      null,
+      "Full path to booclass jar path that will be used on Android runs. If you are using the launcher this will be set for you.");
```

Comment:
nit: bootclass*

Reply:
now that I'm looking at this option, it is not needed. I will also try to get rid of the need for android_native_agent_path option